src/openzaak/conf/includes/base.py

```diff
@@ -350,9 +350,13 @@
 # DJANGO-CSP
 #
 CSP_CONNECT_SRC = CSP_DEFAULT_SRC + ["raw.githubusercontent.com"]
-CSP_SCRIPT_SRC = CSP_SCRIPT_SRC + ["cdnjs.cloudflare.com"]
-CSP_IMG_SRC = CSP_IMG_SRC + ["cdnjs.cloudflare.com", "*.tile.openstreetmap.org"]
-CSP_STYLE_SRC = CSP_STYLE_SRC + ["cdnjs.cloudflare.com"]
+CSP_SCRIPT_SRC = CSP_SCRIPT_SRC + ["cdnjs.cloudflare.com", "cdn.jsdelivr.net"]
+CSP_IMG_SRC = CSP_IMG_SRC + ["cdnjs.cloudflare.com", "tile.openstreetmap.org"]
+CSP_STYLE_SRC = CSP_STYLE_SRC + ["cdnjs.cloudflare.com", "cdn.jsdelivr.net"]
+
+# TODO is there a better way to fix this?
+CSP_INCLUDE_NONCE_IN.remove("script-src")  # error with redoc.
+CSP_INCLUDE_NONCE_IN.remove("style-src")  # error with GISModelAdmin.
```

Comment:
I think the comments for these need to be swapped, it seems that redoc breaks with the style-src nonce and GISModelAdmin with script-src nonce